app/bingey-ui/src/components/WatchlistsControler/WatchlistAdd/WatchlistAdd.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  TextField,
+} from '@material-ui/core';
+import Slide from '@material-ui/core/Slide';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogUI from '@material-ui/core/Dialog';
+import './WatchlistAdd.scss';
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+
+const Transition = React.forwardRef(function Transition(props, ref) {
+  return <Slide direction='up' ref={ref} {...props} />;
+});
+
+export function WatchlistAdd({ onCreateWatchlist, openDrawer }) {
+  const [dialogOpen, setDialogOpen] = useState(false);
```

Comment:
I'd prefer to name this as `isDialogOpen`, it already is quite confusing with `closeDialog` being a handler but named similarly